pkg/didcomm/packer/api.go

```diff
@@ -13,6 +13,22 @@ import (
 	"github.com/hyperledger/aries-framework-go/spi/storage"
 )
 
+const (
```

Comment:
where is EncodingTypeV1 defined?

Reply:
it was already defined in [LegacyPacker](https://github.com/hyperledger/aries-framework-go/blob/main/pkg/didcomm/packer/legacy/authcrypt/authcrypt.go#L24)